masterlist.yaml

```
@@ -8820,6 +8820,19 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507'
+        name: 'Nexus Mods'
+    group: *lateChangesGroup
```

Can you please elaborate on why you want the plugin placed so late? I'm not saying it shouldn't, I'd just appreciate an explanation of your reasoning.

Checking against my own mod list, I only see conflicts with USSEP, which I assume you've addressed based on your mod's description, & Campfire, which already loads before it since it's an ESM. Are there any specific mods in later-loading groups that it should load after?

From my own experience, the fact that Wrye Bash only preserves the conflict winner for VMAD subrecords is a good reason for it to load later than default, but I'm not sure if that's enough for it to load almost dead last.

---

The mod changes the VMAD entry for `MineOreScript` to `WWWMineOreAddons` (which "forwards" MineOreScript's code, so to speak, by _extending_ it) and `ResourceFurnitureScript` to `WWWResourceFurnitureAddons` (same *extending* magic). Any overwrite to any of the wood chopping block base records, or, more likely, one of the ore base records, will prevent the mod from functioning on that particular object, which is much more detrimental than the effects of overwriting other kinds of edits. As such, I do believe it to be important that the mod loads almost-last to ensure that the changes are kept.

Additionally, it is very labor-intensive to create this kind of manual patch. Drag-and-drop is often not enough to cut it, unlike with, say, a Keyword edit. Assuming that Bash forwards my changes, it should be relatively easy for a user to forward any other changes that may need forwarding.

Changes from USSEP are indeed forwarded. I'll have to check out Campfire's changes and, if needed, maybe make a patch.

---

Another thing I forgot to consider with the `Late Fixes & Changes` group is that it loads after `Dynamic Patches`, which contains things like the Bashed Patch & other user-generated patches. `Late Fixes & Changes` should really only be used for mods that absolutely must load after such patches which I don't think applies in this case.

I think the optimal placement would be in `Crafting` since I just checked & [CCOR](https://www.nexusmods.com/skyrimspecialedition/mods/28608) conflicts with a decent chunk of your mod. A patch would obviously be ideal in this case, but loading your mod after it at least prevents it from overwriting half of your mod while still maintaining most of CCOR's features. AFAIK, none of the later-loading groups should contain any mods that would conflict with yours, & we can always revise your mod's position later if necessary.

Note that since both your mod & CCOR have dependents (e.g., patches), you'd also need to add a load after to prevent cyclic interactions:

```suggestion
    group: *craftingGroup
    after: [ 'Complete Crafting Overhaul_Remastered.esp' ]
```